ansible/roles/baremetal/defaults/main.yml

```diff
@@ -86,3 +86,6 @@ virtualenv:
 # directory. This is typically required for modules such as yum and apt which
 # are not available on PyPI.
 virtualenv_site_packages: True
+
+# Whether to remove the AppArmor libvirt profile on Ubuntu hosts.
+apparmor_remove_libvirt_profile: "{{ enable_nova_libvirt_container | bool }}"
```

Comment:
In the upstream version we have: `enable_nova_libvirt_container`

https://review.opendev.org/c/openstack/ansible-collection-kolla/+/830504/3/roles/baremetal/defaults/main.yml#115

Reply:
I see that is in group_vars/all. Ignore me.